packages-ts/gauntlet-terra/src/commands/ledgerKey.ts

```diff
@@ -0,0 +1,60 @@
+import { Key, SimplePublicKey } from '@terra-money/terra.js'
+import LedgerTerraConnector, {ERROR_CODE} from '@terra-money/ledger-terra-js'
+import TransportNodeHid from "@ledgerhq/hw-transport-node-hid"
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
+
+
+export class LedgerKey extends Key {
+    private path: Array<number>
+    private ledgerConnector: LedgerTerraConnector
+
+    constructor(path: Array<number>) {
```

Comment:
Can you make the constructor `private` ? We should force to use the `create` fn to initialize the class

Reply:
done